content/en/docs/languages/net/instrumentation.md

```diff
@@ -7,6 +7,20 @@ description: Instrumentation for OpenTelemetry .NET
 
 {{% docs/languages/manual-intro %}}
 
+{{% alert title="Note" color="info" %}}
+
+On this page you will learn how you can add traces, metrics and logs to your
+code _manually_. But, you are not limited to only use one kind of
+instrumentation: use [automatic instrumentation](/docs/languages/net/automatic/)
+to get started and then enrich your code with manual instrumentation as needed.
+
+Also, for libraries your code depends on, you don't have to write
+instrumentation code yourself, since they might come with OpenTelemetry built-in
+_natively_ or you can make use of
+[instrumentation libraries](/docs/languages/net/libraries/).
```

Comment:
```suggestion
On this page you will learn how you can add traces, metrics and logs to your
code manually. You are not limited to using one kind of instrumentation:
you can also use [automatic instrumentation](/docs/languages/net/automatic/)
to get started and then enrich your code with manual instrumentation as needed.

Also, for libraries your code depends on, you don't have to write
instrumentation code yourself, since they might come with OpenTelemetry
or you can make use of
[instrumentation libraries](/docs/languages/net/libraries/).
```

@svrnm Not sure if the include has the same wording everywhere, but it shouldn't use italics for emphasis.

Reply:
Yes, it is the same wording everywhere (currently js and java) I just changed the references for .Net where necessary.